cms/templates/cms/home_page.html

```diff
@@ -117,14 +117,16 @@
       <div class="row">
         <div class="col-md-8">
           <img src="static/images/logo-mit-white.png" alt="">
```

Comment:
This logo doesn't follow MIT guidelines. Use https://github.com/mitodl/micromasters/blob/fixup/pdpinch/add_footer/static/images/MIT-logo-ltgray-white-72x38.svg instead (may need to be resized)